### ISSUE TITLE
HashLink support?

### ISSUE THREAD

**Issue by chosencharacters** [Tanz0rz/haxe-fmod#7]
Seems to be crashing when trying to export via HashLink in the HaxeFlixel library, the weird part is that it's only during the ReplaceVistaIcon.exe segment which... is very confusing. However, it only happens when this library is used.

```
 - Copying file: export/hl/obj/ApplicationMain.hl -> export/hl/bin/hlboot.dat
 - Renaming file: export/hl/bin/hl.exe -> export/hl/bin/fmod-test.exe
 - Running command: C:/HaxeToolkit/haxe/lib/lime/7,8,0/templates/bin/ReplaceVistaIcon.exe export/hl/bin/fmod-test.exe export/hl/bin/icon.ico 1

Running command: INSTALL

Running command: RUN
 - Changing directory: export/hl/bin/
 - Running command: fmod-test.exe -verbose -livereload
```

Very unusual, happens on the example project as well :(

Does this library do anything regarding icons in any way?

**Comment by Tanz0rz** [comment]
It does not do anything with icons. I haven't used HashLink before. Can you share with me the commands you are running against the example project to make this happen?

**Comment by chosencharacters** [comment]
Yea the command I use directly is

```lime test hl```

HashLink is really useful for quick Desktop testing, it has a compile time
that's usually around 5x - 10x less than regular Desktop :x

**Comment by Tanz0rz** [comment]
I looked into this and it looks like HashLink is a VM made to run Haxe games. So it is not the same thing as targeting `windows` which is something explicitly supported by `haxe-fmod` via C++ code when the `windows` build is targeted. After I complete the core feature/bug work, I will definitely be looking to expand the build targets.

If someone with knowledge around how the HashLink/Windows build target gap can be bridged, it may be an easy thing to do, but do not understand HashLink at all right now (or how lime interacts with it via the `lime test hl` command).

**Comment by chosencharacters** [comment]
Yea that's fair enough, maybe someone else knows HL enough to do this bit.
It's a really cool extension you've made and I'd love to use it but without
HL it's a bit tough to integrate it into my workflow :(

**Comment by chosencharacters** [comment]
Addressed with #8
